styles/facebook/catppuccin.user.less

```
@@ -0,0 +1,475 @@
+  :root {
+    @media (prefers-color-scheme: light) {
+      #catppuccin(@lightFlavor);
+    }
+
+    @media (prefers-color-scheme: dark) {
+      #catppuccin(@darkFlavor);
+    }
+  }
+
+  .__fb-dark-mode:root,
+  .__fb-dark-mode {
+    #catppuccin(@darkFlavor);
+  }
+
+  .__fb-light-mode:root,
+  .__fb-light-mode {
+    #catppuccin(@lightFlavor);
+  }
+
+  #catppuccin(@flavor) {
+    #lib.palette();
+    #lib.defaults();
+
+    html,
+    body,
+    #facebook {
+      background: var(--surface-background);
+      color: var(--primary-text);
+    }
+
+    /* Background */
+    ._95kb,
+    ._95ka {
+      background-color: var(--surface-background);
+    }
+
+    /* Logo */
+    .fb_logo {
+      filter: @accent-filter;
+    }
+
+    /* Title */
+    ._ihd._6s {
+      color: var(--primary-text);
+    }
+
+    /* Subtext */
+    ._8e4l.mbl._6o._6s._mf,
+    ._8eso,
+    ._8esf ._58mk,
+    ._8icy ._6ltj a {
+      color: var(--secondary-text);
+    }
+
+    /* Account Card */
+    ._1gax {
+      background-color: var(--card-background-flat);
+    }
+
+    ._1gbd {
+      box-shadow: none;
+    }
+
+    ._95l5 ._1gaz {
+      background-color: @surface1;
+      color: var(--primary-text);
+    }
+
+    /* icon */
+    .sp_vxiiSmFDFzJ {
+      filter: @accent-filter;
+    }
+
+    /* Login Form */
+    ._6luv {
+      background-color: var(--card-background-flat);
+    }
+
+    /* Inputs */
+    ._55r1 {
+      background-color: var(--input-background);
+      border: 1px solid var(--input-border-color);
+    }
+
+    /* Buttons */
+    ._8icy ._6lth {
+      background-color: @blue;
+      color: var(--primary-text);
+    }
+
+    ._8icy ._6lti {
+      background-color: var(--positive);
+      color: var(--primary-text);
+    }
+
+    /* Input text */
+    ._8icy ._6lux ._6luy {
+      color: var(--primary-text);
+    }
+
+    /* Dividers */
+    ._8icy ._8icz {
+      border-bottom: 1px solid var(--divider);
+    }
+
+    /* Footer */
+    #pageFooter a,
+    ._4ki._6-h > li:first-child {
+      color: var(--secondary-text);
+    }
+
+    #pageFooter .copyright {
+      color: var(--secondary-text);
+    }
+
+    #contentCurve {
+      border-bottom: 1px solid var(--divider);
+    }
+
+    /* Surfaces */
+    --surface-background: @base;
+    --card-background: @mantle;
+    --card-background-flat: @surface0;
+    --background-deemphasized: @surface1;
+    --popover-background: @mantle;
+    --popover-card-background: var(--card-background);
+    --comment-background: @surface0;
+    --comment-footer-background: @surface0;
+    --wash: @surface1;
+    --web-wash: @surface1;
+    --nav-bar-background: @crust;
+    --messenger-card-background: @mantle;
+    --messenger-reply-background: @surface0;
+
+    /* Text & Icons */
+    --primary-text: @text;
+    --secondary-text: @subtext0;
+    --section-header-text: @subtext1;
+    --placeholder-text: @overlay1;
+    --primary-icon: @text !important;
+    --secondary-icon: @overlay1;
```

These are the icons used at the tabs at the top of the page when signed in, they seem too dim to me.
```suggestion
    --secondary-icon: @overlay2;
```